encoding/src/maccommands.rs

```diff
@@ -521,6 +521,23 @@ impl<const N: usize> ChannelMask<N> {
         self.0[index] = value;
     }
 
+
+    /// Enable or disable a specific channel. Recall that LoRaWAN channel numbers start indexing
+    /// at zero.
+    ///
+    /// Improper use of this method could lead to out of bounds panic during runtime!
+    pub fn set_channel(&mut self, channel: usize, set: bool) {
```

Comment:
Did you consider creating a builder for this type? I can be convinced that this case is too simple and having a builder is really an overkill, 🙂